backend/store/playlist.go

```
@@ -0,0 +1,170 @@
+		ORDER BY ps.position ASC
+		`, playlistId)
+	return songs, err
+}
+
+func (ps PlaylistStore) AddSongToPlaylist(playlistId uuid.UUID, songId uuid.UUID) error {
+	fmt.Printf("Adding song to playlist %s", playlistId)
+	tx, err := ps.db.Beginx()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback() 
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)